test/gui/shared/steps/steps.py

```diff
@@ -1466,3 +1462,34 @@ def step(context, username, foldername):
 @Then("credentials wizard should be visible")
 def step(context):
     waitForObject(AccountConnectionWizard.CREDENTIAL_PAGE)
+
+
+@When("user selects configure_synchronization_manually option in advanced section")
+def step(context):
+    newAccount = AccountConnectionWizard()
+    clickButton(waitForObject(newAccount.CONF_SYNC_MANUALLY_RADIO_BUTTON))
+    clickButton(waitForObject(newAccount.NEXT_BUTTON))
+
+
+@Then("wizard add folder sync connection should be visible")
+def step(context):
+    newAccount = AccountConnectionWizard()
+    waitForObject(newAccount.ADD_FOLDER_SYNC_CONNECTION_WIZARD)
+
+
+@Then("wizard select a remote destination folder should be visible")
+def step(context):
+    waitForObject(names.add_Folder_Sync_Connection_groupBox_QGroupBox)
+
+
+@When("user clicks on next button")
+def step(context):
+    newAccount = AccountConnectionWizard()
+    clickButton(waitForObject(newAccount.ADD_FOLDER_SYNC_CONNECTION_NEXT_BUTTON))
+
+
+@When('user selects "|any|" as a remote destination folder')
+def step(context, folderName):
+    newAccount = AccountConnectionWizard()
+    newAccount.OWNCLOUD_AS_A_ROOT_SYNC_DIRECTORY(folderName)
```

Comment:
refactor the function name to comply with the convention that we used. A function name is written in camel case.